runtime/oti/j9nonbuilder.h

```diff
@@ -4709,6 +4709,8 @@ typedef struct J9InternalVMFunctions {
 	U_32 ( *getVMRuntimeState)(struct J9JavaVM *vm);
 	BOOLEAN ( *updateVMRuntimeState)(struct J9JavaVM *vm, U_32 newState);
 	U_32 ( *getVMMinIdleWaitTime)(struct J9JavaVM *vm);
+	void* ( *mem_allocate_memory)(uintptr_t byteAmount) ;
```

Comment:
for clarity I would recommend renaming this to `image_mem_allocate_memory` or something else that distinguishes this from the normal memory allocation.